qt/scientific_interfaces/MultiDatasetFit/MultiDatasetFit.cpp

```diff
@@ -25,6 +28,49 @@ namespace {
 const int zoomToolPage = 0;
 const int rangeToolPage = 1;
 Mantid::Kernel::Logger g_log("MultiDatasetFit");
+
+/// Copy parameter values into a table workspace that is ready for
+/// plotting them against a dataset index.
+void formatParametersForPlotting(const Mantid::API::IFunction &function,
+                                 const std::string &parametersPropertyName) {
+  const auto &mdFunction =
+      dynamic_cast<const Mantid::API::MultiDomainFunction &>(function);
```

Comment:
Would it be worth checking if the cast is successful?

Reply:
Yes. I need to do it. If you run a fit with a single data set it gives you a crash :(